testcode/readzone.c

```diff
@@ -0,0 +1,108 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <stdint.h>
+#include "config.h"
```

Comment:
```suggestion
#include "config.h"
#include <stdio.h>
#include <stdlib.h>
#include <errno.h>
#include <string.h>
#include <unistd.h>

#include <stdint.h>
```
It is necessary to include config.h before the other headers so that it can set system definitions.